README.md

```diff
@@ -0,0 +1,15 @@
+# AvalaraTaxProductOption Module
+[![Build Status](https://travis-ci.org/spryker-eco/avalara-tax-product-option.svg)](https://travis-ci.org/spryker-eco/avalara-tax-product-option)
+[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.2-8892BF.svg)](https://php.net/)
+
+AvalaraTaxProductOption module provides plugins for extending AvalaraTax module functionality with calculating taxes for product options.
```

Comment:
* AvalaraTaxProductOption module provides functionality to calculate taxes for product options.

Reply:
Resolved.